sinks/multi_sink.py

```diff
@@ -9,15 +9,21 @@ class Sinks:
     def __init__(self, sink_commands):
         self.sinks = []
         self.lines_sent = 0
-        for param in sink_commands:
-            print(f'Executing: {param}')
+        for idx, param in enumerate(sink_commands):
+            print(f'Executing {idx + 1}: {param}')
             self.sinks.append(subprocess.Popen(param.split(), stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.STDOUT))
 
     def write_lines(self, lines):
         self.lines_sent += len(lines)
         encoded_lines = [line.encode('utf-8') for line in lines]
         [sink.stdin.writelines(encoded_lines) for sink in self.sinks]
 
+    def print_errors(self):
+        for idx, sink in enumerate(self.sinks):
+            (out, _) = sink.communicate()
```

Comment:
https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate

communicate returns (stdout, stderr), we want to print them both.

Reply:
Ok. I'll add to print stderr too. For my tests I saw error messages were received in stdout but maybe some other sink writes something to stderr.